### ISSUE TITLE
Apps w/ --single-instance will open two tabs when opening into a new tab

### ISSUE THREAD

**Issue by cwensel** [nativefier/nativefier#1209]
**Homework**

- [x] I looked at `nativefier --help` and https://github.com/nativefier/nativefier/blob/master/API.md
- [x] I checked the [Troubleshooting section of the README](https://github.com/nativefier/nativefier/blob/master/README.md#troubleshooting).
- [x] I searched existing issues, open & closed. Yes, my bug is new.
- [x] I'm using the latest version available at https://github.com/nativefier/nativefier/releases

**Bug description**

After upgrading to latest, multiple apps now open two (2) tabs when the link previously would open into a single tab.

An example would be in Gmail, clicking Google Apps upper right, and then selecting an app that would be a new tab will result in two tabs of the same app.

**Steps to reproduce**

```
nativefier 'https://docs.google.com/spreadsheets'  --single-instance
```

**Expected behavior**

One tab will open.

**Actual behavior**

Two tabs will open.

**Debug info**

- Console logs of your `nativefier` build command, with `--verbose` flag
- If the bug happens at app run time, the in-app DevTools console logs (open it with F12)
- Error messages
- Screenshots
- Anything else relevant!

**Context**

 - Nativefier: 43.1.3
 - Node.js: v14.15.4
 - Npm: 7.15.1
 - OS: OS X Catalina 10.15.7

**Comment by TheCleric** [comment]
I'm a little confused on this. What exactly opens two tabs?

On my machine I created an app using your command (except I had to add a user agent as Google sites don't like the default ones for login). After login, I saw only one tab was open. I clicked in the apps, and clicked Gmail. At this point the app now created one new tab for Gmail. Is this the two tabs you mean, or are you saying your issue is you are seeing two new tabs + the original tab (total of 3 tabs)?

**Comment by cwensel** [comment]
yeah, sorry.

If you click a link that would normally open into a new tab (resulting in the first tab and the new tab), you will get three tabs (the first tab, and two copies of the second).

If you click a link that just does a simple navigation in the current tab, it works fine. it's only when the link pops open a new tab.

so say if i'm viewing my calendar, and I mouse over 'google apps', top right, click it, and then choose 'meet', i get two 'meet' tabs.

**Comment by TheCleric** [comment]
> yeah, sorry.
> 
> If you click a link that would normally open into a new tab (resulting in the first tab and the new tab), you will get three tabs (the first tab, and two copies of the second).
> 
> If you click a link that just does a simple navigation in the current tab, it works fine. it's only when the link pops open a new tab.
> 
> so say if i'm viewing my calendar, and I mouse over 'google apps', top right, click it, and then choose 'meet', i get two 'meet' tabs.

Ahh, I see now. It's doing it in some cases, but not all. But it is reproducible for me, so that's at least good (not that we have a bug, but from the "being able to fix it" perspective 😆 ).

**Comment by cwensel** [comment]
glad you can reproduce it.

fwiw, I also notice that when a new tab opens, it opens/inserts immediately to the right, but the title of the end tab changes if there are two or more tabs already open. 

so (left to right, starting with two tabs) you have the first tab on the left, the new second tab with the old title (and new contents), and the third tab with the new title (but former second tab contents).

**Comment by bambootb** [comment]
Same issue happens to me. 

See:
https://monosnap.com/file/drGzOfh9kxYtuhiswnotiQBq2YfvCA

Looking forward for a solution. 

Thank you

**Comment by TheCleric** [comment]
A not as I'm researching this: it doesn't seem to only apply to apps made with `--single-instance`. It appears ALL new tabs are doubled.

**Comment by ronjouch** [comment]
> A not as I'm researching this: it doesn't seem to only apply to apps made with `--single-instance`. It appears ALL new tabs are doubled.

@TheCleric yes, confirmed by dup report https://github.com/nativefier/nativefier/issues/1214 .

**Comment by TheCleric** [comment]
Fix has been coded. Will go out in the next Nativefier release.

**Comment by ronjouch** [comment]
Fix released as 44.0.2, available now on npm. Thanks for the report, and thanks to @TheCleric for the fix!

**Comment by bambootb** [comment]
Thank you so much @TheCleric @ronjouch 👏
